turbopack/crates/turbopack-core/src/module.rs

```diff
@@ -18,6 +20,13 @@ pub trait Module: Asset {
     #[turbo_tasks::function]
     fn ident(&self) -> Vc<AssetIdent>;
 
+    /// The identifier of the [Module] as string. It's expected to be unique and capture
+    /// all properties of the [Module].
+    #[turbo_tasks::function]
+    async fn ident_string(self: Vc<Self>) -> Result<Vc<RcStr>> {
+        Ok(self.ident().resolve().await?.to_string())
```

Comment:
so the `resolve` call here eliminates the resolve local task when calling `to_string`

which saves a spawn? is that the win?

without it you could just `self.ident().to_string()` and remove the 'Result' the 'async' and the 'await'